docs/content/en/docs-dev/concepts/_index.md

```diff
@@ -34,6 +34,7 @@ A collect of resources (containers, services, infrastructure components...) and
 PipeCD supports multiple kinds of applications such as `KUBERNETES`, `TERRAFORM`, `ECS`, `CLOUDRUN`, `LAMBDA`...
 
 ### Environment
+>Deprecated: Please use Labels instead.
```

Comment:
I'm going to add a link to the release note after v0.24.0 will be out, as it would be useful to have a link to how to set the labels.